frontend/app/controllers/sign-in.js

```diff
@@ -0,0 +1,68 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
+/*---- Login Screen ---*/
+
+  loginFailed: false,
+  loginFailedMessage: "Failed to Login",
+  isProcessing: false,
+  isSlowConnection: false,
+  timeout: null,
+
+    success: function(response) {
```

Comment:
Also, check out how things are done at https://github.com/fossasia/open-event-frontend .. That's the biggest ember.js project at FOSSASIA. You could probably borrow loads of stuff from there.